src/detect-lua.c

```diff
@@ -1427,39 +1425,39 @@ static int LuaMatchTest01a(void)
     SigMatchSignatures(&th_v, de_ctx, det_ctx, p1);
 
     if ((PacketAlertCheck(p1, 1))) {
-        printf("sid 1 didn't match on p1 but should have: ");
+        printf("sid 1 didn't match on p1 but should have: "); //
         goto end;
     }
 
     r = AppLayerParserParse(NULL, alp_tctx, &f, ALPROTO_HTTP1, STREAM_TOSERVER, httpbuf2, httplen2);
     if (r != 0) {
-        printf("toserver chunk 2 returned %" PRId32 ", expected 0: ", r);
+        printf("toserver chunk 2 returned %" PRId32 ", expected 0: ", r); //
         goto end;
     }
     /* do detect for p2 */
     SCLogDebug("inspecting p2");
     SigMatchSignatures(&th_v, de_ctx, det_ctx, p2);
 
     if (!(PacketAlertCheck(p2, 1))) {
-        printf("sid 1 didn't match on p2 but should have: ");
+        printf("sid 1 didn't match on p2 but should have: "); //
         goto end;
     }
 
     FlowVar *fv = FlowVarGet(&f, 1);
     if (fv == NULL) {
-        printf("no flowvar: ");
+        printf("no flowvar: "); //
         goto end;
     }
 
     if (fv->data.fv_str.value_len != 1) {
-        printf("%u != %u: ", fv->data.fv_str.value_len, 1);
+        printf("%u != %u: ", fv->data.fv_str.value_len, 1); //
         goto end;
     }
 
     if (memcmp(fv->data.fv_str.value, "2", 1) != 0) {
         PrintRawDataFp(stdout, fv->data.fv_str.value, fv->data.fv_str.value_len);
 
-        printf("buffer mismatch: ");
+        printf("buffer mismatch: "); //
         goto end;
```

Comment:
The `end` label should be removed, now, please, as well as the `result` variable.